test/guest/fedora-atomic.setup

```diff
@@ -18,12 +18,17 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+# Use mac's in name policy, prevents interference from network-scripts
```

Comment:
So, objections temoprarily withdrawn, let's go with @petervo's version and I make a separate PR.